src/Public/Outputs.ps1

```diff
@@ -514,6 +514,10 @@ function Update-PodeWebTextbox
         [string]
         $Id,
 
+        [Alias('Height')]
+        [int]
+        $Size = 4,
```

Comment:
I'd make it so the default here is `0`, so we can skip updating the textarea if no Size is passed.

Reply:
Fixed